File: Lesson 1 - CPP and DSA Intro/GetFactorial/GetFactorial.cs
using System;

int n, factorial = 1, i = 1;

Console.Write("Enter value for n: ");
n = int.Parse(Console.ReadLine());

while (i <= n) {
    factorial *= i;
    i++;
}

Console.WriteLine("Factorial: " + factorial);

/*
    To run this code, use the following commands in the terminal:
    cd Lesson 1 - CPP and DSA Intro
    cd get factorial
    dotnet-script GetFactorial.cs

*/

File: Lesson 1 - CPP and DSA Intro/GetFactorial/GetFactorial.java
import java.util.Scanner;

public class GetFactorial {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n, factorial = 1, i = 1;

        System.out.print("Enter value for n: ");
        n = scanner.nextInt();

        while (i <= n) {
            factorial *= i;
            i++;
        }

        System.out.println("Factorial: " + factorial);
        scanner.close();
    }
}
